pkg/unvanquished_src.dpkdir/bots/default_humans.bt

```diff
@@ -2,6 +2,50 @@ selector
 {
 	behavior use_medkit
 
+	sequence
+	{
+		condition levelTime >= 60000
```

Comment:
This stuff should be a block-style condition like `condition <expr> { ... }` rather than a sequence of condition then action, so that once there is another builder on the team the bot gives up on trying to become builder (see https://wiki.unvanquished.net/wiki/Formats/Behavior_tree#The_condition_keyword). Same goes for other team's bt